tests/datasets/test_fsd50k.py

```diff
@@ -0,0 +1,193 @@
+
+from soundata import annotations
+from soundata.datasets import fsd50k
+
+TEST_DATA_HOME = "tests/resources/sound_datasets/fsd50k"
+
+
+def test_clip():
+    default_clipid = "64760"
+    dataset = fsd50k.Dataset(TEST_DATA_HOME)
+    clip = dataset.clip(default_clipid)
+
+    expected_attributes = {
+        "audio_path": "tests/resources/sound_datasets/fsd50k/FSD50K.dev_audio/64760.wav",
+        "clip_id": "64760",
+        "sub_set": "dev",
+    }
+
+    expected_property_types = {
+        "audio": tuple,
+        "labels": annotations.Tags,
+        "split": str,
+        "description": str,
+    }
+
+    run_clip_tests(clip, expected_attributes, expected_property_types)
+
+
+def test_load_audio():
+    dataset = fsd50k.Dataset(TEST_DATA_HOME)
+    clip = dataset.clip("64760")
+    audio_path = clip.audio_path
+    audio, sr = fsd50k.load_audio(audio_path)
+    assert sr == 44100
+    assert type(audio) is np.ndarray
+    assert len(audio.shape) == 1  # check audio is loaded as mono
+
+
+def test_to_jams():
+    default_clipid = "64760"
+    dataset = fsd50k.Dataset(TEST_DATA_HOME)
+    clip = dataset.clip(default_clipid)
+    jam = clip.to_jams()
+
+    # Validate fsd50k jam schema
+    assert jam.validate()
+
+    # Validate Tags
+    tags = jam.search(namespace="tag_open")[0]["data"]
+    assert len(tags) == 5
+    assert [tag.time for tag in tags] == [0.0, 0.0, 0.0, 0.0, 0.0]
+    assert [tag.duration for tag in tags] == [
+        1.7143083900226757,
+        1.7143083900226757,
+        1.7143083900226757,
+        1.7143083900226757,
+        1.7143083900226757,
+    ]
+    assert [tag.value for tag in tags] == [
+        "Electric_guitar",
+        "Guitar",
+        "Plucked_string_instrument",
+        "Musical_instrument",
+        "Music",
+    ]
+    assert [tag.confidence for tag in tags] == [1.0, 1.0, 1.0, 1.0, 1.0]
+
+    # validate metadata
+    assert jam.file_metadata.duration == 1.7143083900226757
+    assert jam.file_metadata.title == "guitarras_63.wav"
+    assert jam.sandbox.mids == [
+        "/m/02sgy",
+        "/m/0342h",
+        "/m/0fx80y",
+        "/m/04szw",
+        "/m/04rlf",
+    ]
+    assert jam.sandbox.split == "train"
+    assert jam.sandbox.description == "electric guitar"
+    assert jam.sandbox.tags == [
+        "electric",
+        "guitar",
+    ]
+    assert jam.sandbox.license == "http://creativecommons.org/licenses/sampling+/1.0/"
+    assert jam.sandbox.uploader == "casualsamples"
+    assert jam.sandbox.pp_pnp_ratings == {"/m/02sgy": [1.0, 1.0]}
+    assert jam.annotations[0].annotation_metadata.data_source == "soundata"
+
+
+def test_labels():
+    default_clipid = "64760"
+    dataset = fsd50k.Dataset(TEST_DATA_HOME)
+    clip = dataset.clip(default_clipid)
+    labels = clip.labels
+    assert labels.labels == [
+        "Electric_guitar",
+        "Guitar",
+        "Plucked_string_instrument",
+        "Musical_instrument",
+        "Music",
+    ]
+
+
+def test_dev_metadata():
```

Comment:
I guess you're testing them separately because they have different formats?

Reply:
yes they were, but not anymore actually! I will merge it into a single function. Thanks for the suggestion:)